tests/core/gui/test_invite_user.py

```diff
@@ -0,0 +1,66 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
```

Comment:
should move this test into `core/gui/users_widget`

Reply:
i moved the test into `core/gui/users_widget/test_invite.py`